moveit_ros/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_ros_integration_tests)
```

Comment:
That true, of course. However, the name underlines, that those integrations tests require a ROS environment ;-)